rollup.config.js

```diff
@@ -1,6 +1,7 @@
 import resolve from '@rollup/plugin-node-resolve';
 import commonjs from '@rollup/plugin-commonjs';
 import typescript from '@rollup/plugin-typescript';
+import json from '@rollup/plugin-json';
```

Comment:
import it for?

Reply:
As is shown in the PR desc, an error occurred in build process.

Reply:
Okay, `@stdlib/stats` needs this plugin.